ch02.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[ch02_getting_started]]
+== Getting Started
+
+// TODO intro paragraph - no empty headings!
+
+==== Choosing a Lightning Network Wallet
+
+* full nodes (c-lightning, eclair, lnd) + remote controls
+* phone / desktop wallets (SPV clients)
+* custodial services / wallets?
+// Mastering bitcoin also had a section about custodial web wallets. So it might be fair to include them.
+
+==== Quick Start
+
+* We start with a mobile wallet like eclair
```

Comment:
BTW it does not and will never run on iOS because of the JVM

Reply:
If we want to cover both platforms, then there're few wallets on iOS that we can choose from.